studio/components/interfaces/Organization/OAuthApps/AuthorizeRequesterDetails.tsx

```diff
@@ -18,13 +73,73 @@ const AuthorizeRequesterDetails = ({ icon, name, domain }: AuthorizeRequesterDet
+      </div>
+      <div>
+        <h2>Permissions</h2>
+        <div className="pt-2">
+          <ScopeSection
+            description="access to all project's auth configurations and SSO providers"
+            readScope={OAuthScope.AUTH_READ}
+            writeScope={OAuthScope.AUTH_WRITE}
+            allScopes={scopes}
+          />
+          <ScopeSection
+            description="access to all project's Postgres configurations, SQL snippets, SSL enforcement configurations and Typescript schema types."
+            readScope={OAuthScope.DATABASE_READ}
+            writeScope={OAuthScope.DATABASE_WRITE}
+            allScopes={scopes}
+          />
+          <ScopeSection
+            description="access to all project's custom domains and vanity subdomains."
+            readScope={OAuthScope.DOMAINS_READ}
+            writeScope={OAuthScope.DOMAINS_WRITE}
+            allScopes={scopes}
+          />
+          <ScopeSection
+            description="access to all project's edge functions."
+            readScope={OAuthScope.EDGE_FUNCTIONS_READ}
+            writeScope={OAuthScope.EDGE_FUNCTIONS_WRITE}
+            allScopes={scopes}
+          />
+          <ScopeSection
+            description="access to all project's environments/branches."
+            readScope={OAuthScope.ENVIRONMENT_READ}
+            writeScope={OAuthScope.ENVIRONMENT_WRITE}
+            allScopes={scopes}
+          />
+          <ScopeSection
+            description="access to the organization and all its members."
+            readScope={OAuthScope.ORGANIZATIONS_READ}
+            writeScope={OAuthScope.ORGANIZATIONS_WRITE}
+            allScopes={scopes}
+          />
+          <ScopeSection
+            description="access to all project's metadata, its upgrade status, network restrictions and network bans."
+            readScope={OAuthScope.PROJECTS_READ}
+            writeScope={OAuthScope.PROJECTS_WRITE}
+            allScopes={scopes}
+          />
+          <ScopeSection
+            description="access to all project's postgREST configurations."
+            readScope={OAuthScope.REST_READ}
+            writeScope={OAuthScope.REST_WRITE}
+            allScopes={scopes}
+          />
+          <ScopeSection
+            description="access to all project's API keys, secrets and pgsodium configurations."
+            readScope={OAuthScope.SECRETS_READ}
+            writeScope={OAuthScope.SECRETS_WRITE}
+            allScopes={scopes}
+          />
+        </div>
       </div>
+
       <p className="text-sm text-foreground-light">
-        {name} ({domain}) is requesting API access to an organization. The application will be able
-        to{' '}
-        <span className="text-amber-1200">
-          read and write the organization's settings and all of its projects.
-        </span>
+        Please note, the shown scopes will apply for the{' '}
+        <span className="text-amber-1200">selected organization and all of its projects.</span>
```

Comment:
lets also use text-amber-900 here to make it more visually constrasting

```suggestion
        <span className="text-amber-900">selected organization and all of its projects.</span>
```